packages/svelte/src/compiler/phases/1-parse/remove_typescript_nodes.js

```diff
@@ -73,6 +73,9 @@ const visitors = {
 	TSNonNullExpression(node, context) {
 		return context.visit(node.expression);
 	},
+	TSTypeAnnotation() {
+		return b.empty;
```

Comment:
mhhhhm can we really just return `b.empty`? I worry that it would result in a `;` in the output at an invalid place. Then again, esrap seems to handle it fine / just ignore it? Either way, I would feel better if the fix was to instead go up to the parent, and set the `typeAnnotation` property to `null` on it

Reply:
I think it handle this just fine but if you prefer...also wouldn't returning null be the same as replacing `typeAnnotation` on the parent? `zimmerframe` should replace everything that you return right?

Reply:
Uh no i was wrong

Reply:
Let's wait on a few more opinions on this

Reply:
Are we sure doing this
```ts
TSTypeAnnotation(_, context) {
	const parent = context.path.at(-1);
	if (parent) {
		parent.typeAnnotation = null;
	}
},
```
is a better fix? This kinda assume that TSTypeAnnotation can only be in a `typeAnnotation` field which is probably `true` and i was making this assumption myself in the original fix...but the current fix doesn't suffer this problem and it doesn't seem to affect the output at all.

Reply:
> Let's wait on a few more opinions on this

Uh sorry i was writing the message while you were writing this.

Reply:
Let's go with this solution, but add a comment
```suggestion
		// This isn't correct, strictly speaking, and could result in invalid ASTs (like an empty statement within function parameters),
		// but esrap, our printing tool, just ignores these AST nodes at invalid positions, so it's fine
		return b.empty;
```